generic/tests/netperf.py

```diff
@@ -130,6 +130,13 @@ def mtu_set(mtu):
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
+        if params.get("netdst_nic1") in process.system_output(
+                "ovs-vsctl list-br", ignore_status=True, shell=True).decode():
+            ovs_list = "ovs-vsctl list-ports %s" % params["netdst_nic1"]
+            ovs_port = process.system_output(ovs_list,
+                                             shell=True).decode().splitlines()
+            target_ifaces = target_ifaces + \
+                params.object_paramsts("netdst_nic1") + ovs_port
```

Comment:
There is typo `paramsts`. And here should not use `object_params()`, that's to parse params by suffix, you just want a list of `netdest_nic1`, which should use `objects()`

Reply:
I am curious of adding ovs ports in the target_ifaces, do we need perform `ifconfig ${Interface} mtu ${size}` on ovs ports? @xiywang Would you help confirm this? Thanks.

Reply:
@vivianQizhu Only if you need MTU other than 1500(default value).

Reply:
@vivianQizhu, updated, and  I found previous test by using "params.objects("netidst_nic1")" .